dashboard/app/views/levels/show.html.haml

```diff
@@ -28,7 +28,7 @@
   -# A teacher is trying to view a student's progress on a level they have not started.
   = render partial: 'levels/not_started'
 - elsif @level.is_a?(Blockly) && @level.embed == 'true' && !@edit_blocks
-  #title
+  #title{style: 'text-align: center'}
```

Comment:
Note that after I merge this PR https://github.com/code-dot-org/code-dot-org/pull/5335/files, this won't even be shown in Applab. Should it be shown + centered?